cspatients/views.py

```diff
@@ -149,7 +149,7 @@ def entrycompleted(request):
             )
         except PatientEntry.DoesNotExist:
             return HttpResponse(status=404)
-        patiententry.completion_time = datetime.now()
+        patiententry.delivery_time = datetime.now()
```

Comment:
Use `timezone` instead of `datetime`. See https://docs.djangoproject.com/en/2.0/topics/i18n/timezones/#naive-and-aware-datetime-objects